### PR TITLE
[ETFE-2406] added task list

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: app/controllers/sections/info/DispatchDetailsController.scala [hunk 1]
```diff
@@ -43,38 +43,92 @@ class DispatchDetailsController @Inject()(
...
+            formWithErrors,
+            controllers.sections.info.routes.DispatchDetailsController.onPreDraftSubmit(request.ern, mode),
+            mode
+          ),
+        value =>
+          savePreDraftAndRedirect(DispatchDetailsPage(), value, mode)
+      )
+    }
+
+  def onPageLoad(ern: String, draftId: String): Action[AnyContent] =
```
Comment (ericohwotu) [inline app/controllers/sections/info/DispatchDetailsController.scala:78]: do we need a mode?

File: app/models/sections/transportUnit/TransportUnitsAddToListModel.scala [hunk 2]
```diff
@@ -46,9 +46,9 @@ object TransportUnitsAddToListModel extends Enumerable.Implicits {
 
     Seq(
       radioItem(Yes, 0),
-      radioItem(MoreToCome, 1),
+      radioItem(NoMoreToCome, 1),
       orDivider,
-      radioItem(NoMoreToCome, 2)
+      radioItem(MoreToCome, 2)
```
Comment (ericohwotu) [inline app/models/sections/transportUnit/TransportUnitsAddToListModel.scala:51]: has the order changed?

File: app/controllers/sections/info/DispatchDetailsController.scala (hunk 1 above)
Comment (james-work-account) [reply app/controllers/sections/info/DispatchDetailsController.scala:78]: No, you're only ever going to hit this post-draft from the CYA page so you'll only ever be changing it (mode is inferred to always be CheckMode)

File: app/models/sections/transportUnit/TransportUnitsAddToListModel.scala (hunk 2 above)
Comment (james-work-account) [reply app/models/sections/transportUnit/TransportUnitsAddToListModel.scala:51]: Screenshot from prototype:

image

More to come should've been the last one, the underlying numbers were just the wrong way around even though they looked the same 🙂

Comment (ericohwotu) [reply app/models/sections/transportUnit/TransportUnitsAddToListModel.scala:51]: It was the last one in the demo, not sure why / when it changed

Comment (ericohwotu) [reply app/models/sections/transportUnit/TransportUnitsAddToListModel.scala:51]: Ahhh

**Review by ericohwotu — APPROVED** [review]

**Pull request merged**